README.md

```diff
@@ -545,6 +547,12 @@ The `--dockerfile` option makes it possible to build a new minified image direct
 
 The `--use-local-mounts` option is used to choose how the Slim sensor is added to the target container and how the sensor artifacts are delivered back to the master. If you enable this option you'll get the original Slim app behavior where it uses local file system volume mounts to add the sensor executable and to extract the artifacts from the target container. This option doesn't always work as expected in the dockerized environment where Slim itself is running in a Docker container. When this option is disabled (default behavior) then a separate Docker volume is used to mount the sensor and the sensor artifacts are explicitly copied from the target container.
 
+### `DEBUG` COMMAND OPTIONS
+
```

Comment:
It looks like you lost the `--debug-image` flag. We definitely need to have it here.

Reply:
Ahh, my bad! I think I lost it while editing!